src/main/java/org/javacord/bot/commands/MavenCommand.java

```diff
@@ -20,7 +20,7 @@ public void onCommand(TextChannel channel) {
                            "<dependency>\n" +
                            "    <groupId>org.javacord</groupId>\n" +
                            "    <artifactId>javacord</artifactId>\n" +
-                           "    <version>3.0.0</version>\n" + // TODO Always use the latest version
+                                  "    <version>3.0.1</version>\n" + // TODO Always use the latest version
```

Comment:
While not using the latest Javacord version from some request, at least use `Javacord.VERSION` in the meantime? But keep the TODO or better make an issue instead or additionally.